tools/caffe_translator/README.md

```diff
@@ -0,0 +1,78 @@
+# Caffe Translator
+Caffe Translator is a source code translator that translates Caffe code into MXNet Python code. Note that this is different from the Caffe to MXNet model converted which is available [here](https://github.com/apache/incubator-mxnet/tree/master/tools/caffe_converter).
```

Comment:
Consider naming it: "Caffe Code Translator" since a Caffe Translator exists in Caffe2 and it converts Caffe models to Caffe2 and doesn't have similar functionality to what is being presented here. I had to read this doc several times for it to sink in that these things were very different indeed. It's like a menu item called milkshake, but instead of ice cream, you get to drink some milk and someone grabs your head from behind and shakes it vigorously. If that's in the small text you might be surprised at the outcome.

Reply:
Thanks for this feedback, Aaron. I’ve made changes in the second paragraph to include several samples of both input and output so that it is clear what the tool does. 

A name change will be hard to do this close to the release. But we can definitely discuss this post-release and make a name change if necessary.